scripts/stackstorm.js

```diff
@@ -252,10 +251,6 @@ module.exports = function(robot) {
       .catch(function (err) {
         var error_msg = 'Failed to retrieve commands from "%s": %s';
         robot.logger.error(util.format(error_msg, env.ST2_API_URL, err.message));
-        // Exit in the first time when loadCommands is called with invalid ST2_API_URL, ST2_API_KEY or ST2_AUTH_TOKEN.
-        if (opts.exitOnFailure) {
-          logErrorAndExit(err);
```

Comment:
Our previous conversation was around this line.
So please check for `Unauthorized` and include `logErrorAndExit` here

Reply:
`Unauthorized` is appeared when called with loadCommand and it is only called `st2chatops` initialization. If we want capture authentication issue all the time, it will be moved  `start` function